daemon/src/wallet.rs

```diff
@@ -22,13 +22,49 @@ use maia::TxBuilderExt;
 use model::Timestamp;
 use model::TxFeeRate;
 use model::WalletInfo;
+use statrs::statistics::Distribution;
+use statrs::statistics::Max;
+use statrs::statistics::Median;
+use statrs::statistics::Min;
 use std::collections::HashSet;
 use std::time::Duration;
 use tokio::sync::watch;
 use tokio_tasks::Tasks;
 use xtra_productivity::xtra_productivity;
 use xtras::SendInterval;
 
+static BALANCE_GAUGE: conquer_once::Lazy<prometheus::Gauge> = conquer_once::Lazy::new(|| {
+    prometheus::register_gauge!("balance", "The sum of available UTXOs in the wallet.").unwrap()
+});
+static NUM_UTXO_GAUGE: conquer_once::Lazy<prometheus::Gauge> = conquer_once::Lazy::new(|| {
+    prometheus::register_gauge!("num_utxos", "The number of available UTXOs in the wallet.")
+        .unwrap()
+});
+static MEDIAN_UTXO_VALUE_GAUGE: conquer_once::Lazy<prometheus::Gauge> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_gauge!("median_utxo_value", "The median UTXO.").unwrap()
```

Comment:
The second field of this macro is the unit. Can we add that this is in satoshi?

Reply:
I understand that it is a general help text but yes can certainly add in the unit!